src/components/backend-ai-usersettings-general-list.ts

```diff
@@ -184,6 +184,9 @@ export default class BackendAiUsersettingsGeneralList extends BackendAIPage {
           --mdc-theme-surface: white;
           --mdc-list-vertical-padding: 5px;
           --mdc-list-side-padding: 25px;
+          /* Need to be set when fixedMenuPosition attribute is enabled */
+          --mdc-menu-max-width: 400px;
+          --mdc-menu-min-width: 400px;
```

Comment:
Please make the width the same.
<img width="508" alt="image" src="https://user-images.githubusercontent.com/28584164/178412699-6d6e400c-2d47-44f4-b8e6-07e47f9f2b73.png">

Reply:
hmm... holding.
Let me see... I think that change some css properties  
Because, select "French" or "Russian" then...

Reply:
When I selected "French" language
<img width="452" alt="image" src="https://user-images.githubusercontent.com/20002/179683672-d3ec74d7-791d-4854-bb07-c70508643e4c.png">

When I selected "Russian" language
<img width="446" alt="image" src="https://user-images.githubusercontent.com/20002/179683748-498f5659-2c4e-4d00-b60b-9944d11c7709.png">

When I selected "Mongolian" language
<img width="425" alt="image" src="https://user-images.githubusercontent.com/20002/179683803-24d1e99d-b349-494c-9cec-c075ee949e47.png">

When I selected "Indonesian" language
<img width="427" alt="image" src="https://user-images.githubusercontent.com/20002/179683918-bce5bfa8-f396-4970-94d2-ea725f8de7b4.png">

Reply:
expand width (140 px to 160px) and change  "..." to new line processing